packages/query/package.json

```diff
@@ -81,6 +83,11 @@
   "peerDependencies": {
     "react": "*"
   },
+  "pnpm": {
+    "overrides": {
+      "p-queue": "7.4.1"
```

Comment:
I am rolling back `p-queue` a version and setting it to `7.4.1` as otherwise queries will throw an error for undefined `options.signal?.throwIfAborted` in React Native code. 

I tried to fix this to no avail and discovered the queries worked OK if I bypass the priority queue. 

I tracked back their releases and found they introduced this change [here](https://github.com/sindresorhus/p-queue/compare/v%20...v8.0.0) . On their README it says the project is feature complete and that they don't plan any further development / offer support so I think it's just easier to roll back for now.

TBD - I need to test that this will work in the `extension`